services/docker/compose.go

```diff
@@ -32,6 +32,18 @@ func composeArgs(args ...string) []string {
 	return append(composeFile, args...)
 }
 
+// CreateDatabaseVolume will create a database volume if it doesn't already exist
+func CreateDatabaseVolume() {
+	n := "tok_" + conf.GetConfig().Tokaido.Project.Name + "_mysql_database"
+	o := utils.BashStringCmd(`docker volume ls | grep ` + n)
+	if o == n {
+		utils.DebugString("existing database volume found, not creating a new one")
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

> Error strings should not be capitalized

We can split hairs over this one, since this debug message isn't an error. But I like the format of lowercase for non-standard user messages. Personal preference.

Reply:
I don't think we're doing that anywhere else? Different rules for debug messages?

Reply:
Yeah fair enough. We don't do that anywhere else in Tokaido. 

Ginza and Shinjuku follow this approach however, so you and I should probably have a discussion about a global standard. 

But for now, I've updated the text